dev/serializing_util.h

```diff
@@ -197,8 +197,9 @@ namespace sqlite_orm {
             auto& context = get<2>(tpl);
 
             constexpr std::array<const char*, 2> sep = {", ", ""};
-            for(size_t i = 0, first = true; i < args.size(); ++i) {
-                ss << sep[std::exchange(first, false)] << serialize(args[i], context);
+            bool first = true;
+            for(auto it = args.begin(); it != args.end(); ++it) {
```

Comment:
Why not use `for(const auto& arg : args)`?

Reply:
cause I have weird habits left from old C++ standards. Just fixed. Thanks